R/gips_class.R

```diff
@@ -1523,7 +1526,9 @@ summary.gips <- function(object, ...) {
       number_of_observations = attr(object, "number_of_observations"),
       was_mean_estimated = attr(object, "was_mean_estimated"),
       delta = attr(object, "delta"),
-      D_matrix = attr(object, "D_matrix")
+      D_matrix = attr(object, "D_matrix"),
+      AIC = suppressWarnings(AIC.gips(object)), # warning for NA
```

Comment:
Normally, the explicit call to AIC.gips would not be necessary; AIC would suffice.
It raises warnings in almost-singular case, right? Perhaps we could supress only this particular warning and pass all others

Reply:
Done